source/features/quick-review-comment-deletion.tsx

```diff
@@ -33,7 +33,7 @@ function init(signal: AbortSignal): Deinit {
 	delegate(document, '.rgh-review-comment-delete-button', 'click', onButtonClick, {signal});
 	delegate(document, '.rgh-quick-comment-edit-button', 'click', onEditButtonClick, {signal});
 
-	return observe('.review-comment > .unminimized-comment form:not(.js-single-suggested-change-form) .js-comment-cancel-button:not(.rgh-delete-button-added)', {
+	return observe('.review-comment form.js-comment-update:not(.js-single-suggested-change-form) .js-previewable-comment-form .form-actions .js-comment-cancel-button:not(.rgh-delete-button-added)', {
```

Comment:
Thanks for the PR!

I think this selector can be greatly shortened now.

Testing:

```suggestion
a
```